### PR TITLE
Create hacktoberfest.py

### PR DESCRIPTION
**Pull request by pavitra1129**
this program will print hacktoberfest 2022 for 1000 times

### CODE REVIEW

**Comment by its0din-ai** [comment]
We Will merge this If you can add an Indexing,
like :
0. Hacktoberfest
1. Hacktoberfest
...

**Review by its0din-ai — CHANGES_REQUESTED** [review]
Add for loops Indexing

**Comment by its0din-ai** [comment]
We will mark this as Spam.

**Comment by pavitra1129** [comment]
here is the code you wanted sorry for responding late but here is the code you wanted please remove the spam label
for i in range(0,1001):
    p = str(i)
    print(p+'''. hacktoberfest 2022''')

**Comment by pavitra1129** [comment]
please remove the spam label

**Review by its0din-ai — CHANGES_REQUESTED** [review]
modify this

**Comment by pavitra1129** [comment]
sir is this ok

**Comment by pavitra1129** [comment]
or you want any other change

File: Python/hacktoberfest.py [hunk 1]
```diff
@@ -0,0 +1 @@
+ print("hacktoberfest 2022"*1000)
```
Comment (its0din-ai) [inline Python/hacktoberfest.py]: Ask for user input, then loop the "Hacktoberfest2022" according to the input

**Review by its0din-ai — APPROVED** [review]

**Pull request merged**